build-system/task-runner/amp-task-runner.js

```diff
@@ -106,9 +114,9 @@ async function runTask(taskName, taskSourceFileName, taskFunc) {
 /**
  * Prints an error if the task file and / or function are invalid, and exits.
  * @param {string} taskSourceFileName
- * @param {string?} taskFuncName
+ * @param {string=} taskFuncName
  */
-function handleInvalidTaskError(taskSourceFileName, taskFuncName) {
+function handleInvalidTaskError(taskSourceFileName, taskFuncName = undefined) {
```

Comment:
This should not be needed
```suggestion
function handleInvalidTaskError(taskSourceFileName, taskFuncName) {
```